packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -187,6 +187,31 @@ QUnit.module('Chat', moduleConfig, () => {
                 assert.strictEqual(onMessageSend.callCount, 1);
             });
 
+            QUnit.test('should be called on enter key when some text was entered', function(assert) {
```

Comment:
Tests should be placed in message_box part

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/28003/commits/6f237803a45dbd8389f83f0a6cbab4ee964a88fb)